Sources/Core/UI/PreviewHelper.swift

```diff
@@ -102,7 +102,7 @@ internal class PreviewHelper {
             fileURL.appendPathComponent("master.m3u8")
             completion(self.processFileDownload(to: fileURL, result: .success(())))
         }
-        else if supportedFileFormat.contains(fileURL.pathExtension) {
+        else if supportedFileFormat.contains(fileURL.pathExtension.lowercased()) {
```

Comment:
I think that according to this change we should also change `BMPf` to` bmpf` in `otherFileFormat` property (line 22). Otherwise it won't be recognized as a supported format.

Reply:
I've already made this change in a second commit in this PR